website/docs/Ecosystem.md

```diff
@@ -10,4 +9,12 @@ This page lists libraries that have integrations with Autogen for LLM applicatio
 
 MemGPT enables LLMs to manage their own memory and overcome limited context windows. You can use MemGPT to create perpetual chatbots that learn about you and modify their own personalities over time. You can connect MemGPT to your own local filesystems and databases, as well as connect MemGPT to your own tools and APIs. The MemGPT + AutoGen integration allows you to equip any AutoGen agent with MemGPT capabilities.
 
-- [MemGPT + AutoGen documentation with code examples](https://memgpt.readthedocs.io/en/latest/autogen/)
+- [MemGPT + AutoGen Documentation with Code Examples](https://memgpt.readthedocs.io/en/latest/autogen/)
+
+## Semantic Kernel + AutoGen
+
+![Agent Chat Example](img/ecosystem_SemanticKernel.png)
+
+Semantic Kernel is an SDK that integrates Large Language Models (LLMs) like OpenAI, Azure OpenAI, and Hugging Face with conventional programming languages like C#, Python, and Java. Semantic Kernel achieves this by allowing you to define plugins that can be chained together in just a few lines of code. With Semantic Kernel, you can easily bring Plugins to agents so they can do more than just talk.
+
+- [Semantic Kernel + AutoGen Quick Tutorial](https://devblogs.microsoft.com/semantic-kernel/autogen-agents-meet-semantic-kernel/)
```

Comment:
I don't like: "With Semantic Kernel, you can easily bring Plugins to agents so they can do more than just talk."
It can be misleading. Even without SK, AutoGen agents do more than just talk.
I'd suggest removing this example for now.

Reply:
resolved